.changeset/moody-lies-fry.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/icons': patch
+'@twilio-paste/core': patch
```

Comment:
We don't patch core for icon changes.  Icons is the only thing that doesn't also update core.

Reply:
ah, good to know! Thanks!